### PR TITLE
Add Integration tests

### PR DESCRIPTION
**Pull request by chingor13**
Adds an integration test to Circle CI

* Runs memcache and mysql servers for tests
* Generates and tests a wordpress instance
* Generates and tests a laravel instance (with Eloquent using mysql)

Note: to test the current branch, we replace the repo/branch in the composer.json before installing dependencies.

See #120

### CODE REVIEW

File: tests/integration/curl/CurlTest.php [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    {
+        if (!extension_loaded('opencensus')) {
+            $this->markTestSkipped('Please enable the opencensus extension.');
+        }
+        opencensus_trace_clear();
+    }
+
+    public function testCurlExec()
+    {
+        $url = 'https://www.google.com/';
```
Comment (jcchavezs) [inline tests/integration/curl/CurlTest.php]: I think in here we could avoid this Google dependency. Would you consider to use something like https://github.com/jcchavezs/httptest-php

Comment (chingor13) [reply tests/integration/curl/CurlTest.php]: Yes, I think that would work.

Comment (chingor13) [reply tests/integration/curl/CurlTest.php]: We'll skip this for now: https://github.com/jcchavezs/httptest-php/issues/5

File: src/Trace/Integrations/Eloquent.php [hunk 2]
```diff
@@ -77,12 +77,9 @@ public static function load()
         });
 
         // public function delete()
-        opencensus_trace_method(Model::class, 'delete', function ($model, $query) {
```
Comment (chingor13) [inline src/Trace/Integrations/Eloquent.php]: This was broken and caught by integration tests - there is no parameter for query.

File: tests/integration/guzzle6/tests/bootstrap.php [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<?php
+
+require __DIR__ . '/../vendor/autoload.php';
```
Comment (tmatsuo) [inline tests/integration/guzzle6/tests/bootstrap.php]: If this is the only content, maybe you can just specify the autoload.php in the phpunit config file

File: tests/integration/memcached/tests/bootstrap.php [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<?php
+
+require_once __DIR__ . '/../vendor/autoload.php';
```
Comment (tmatsuo) [inline tests/integration/memcached/tests/bootstrap.php]: ditto

File: tests/integration/guzzle6/tests/bootstrap.php (hunk 3 above)
Comment (chingor13) [reply tests/integration/guzzle6/tests/bootstrap.php]: done.

File: tests/integration/memcached/tests/bootstrap.php (hunk 4 above)
Comment (chingor13) [reply tests/integration/memcached/tests/bootstrap.php]: done.

File: tests/integration/curl/CurlTest.php (hunk 1 above)
Comment (jcchavezs) [reply tests/integration/curl/CurlTest.php]: @chingor13 I just opened a PR addressing this issue: https://github.com/jcchavezs/httptest-php/pull/6

**Pull request merged**